Correctly update the statusbar when searching.

Calling Clear() directly on playlistModel should really be disallowed...

 PlayerInterface.cs |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)


Index: src/PlayerInterface.cs
===================================================================
RCS file: /cvs/gnome/banshee/src/PlayerInterface.cs,v
retrieving revision 1.168
diff -u -p -r1.168 PlayerInterface.cs
--- src/PlayerInterface.cs	21 Mar 2006 22:16:22 -0000	1.168
+++ src/PlayerInterface.cs	25 Mar 2006 20:46:37 -0000
@@ -1315,7 +1315,7 @@ namespace Banshee
                 return;
             }
             
-            playlistModel.Clear();
+            playlistModel.ClearModel();
             
             if(!searchEntry.IsQueryAvailable) {
                 playlistModel.ReloadSource();
